hera_pspec/tests/test_pstokes.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 import os, sys
 from hera_pspec.data import DATA_PATH
-from .. import pstokes 
+from .. import pstokes
```

Comment:
My personal opinion is to not use relative imports in the testing suite, so I think from hera_pspec import pstokes is better here (I know that wasn't what was added in this PR but might as well address it while we are here)

Reply:
I was having trouble getting the correct coverage registered without relative imports (when I don't use relative imports, for some reason running something like `pytest --cov-report html` results in no coverage being registered. Maybe something else is configured incorrectly that is causing this. I had the same problem migrating to pytest in `uvtools` recently.

Reply:
I see, yeah that's weird--i get coverage locally on my machine.

I think this may not be the place to institute this change, but I do think non-relative imports seems to be the standard for test files (see pyuvdata and astropy, for example). let's come back to this later, I'll make an issue

Reply:
sounds good!